server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/DeletedMailsDAO.java

```diff
@@ -53,6 +53,7 @@
 
     private PreparedStatement prepareInsert(Session session) {
         return session.prepare(insertInto(TABLE_NAME)
+            .ifNotExists()
```

Comment:
I doubt this change will perform better on a cluster. We should run benchs on a Cassandra cluster IMO.